charlib/cli/utils.py

```
@@ -23,7 +23,17 @@ def find_config(config_path, quiet=True):
     for file in find_yaml_files(config_path):
         try:
             with open(file, 'r') as f:
-                config = ConfigFile.validate(yaml.safe_load(f))
+                config = yaml.safe_load(f)
+                # Allow "settings" and "cells" keys to point to other yaml files
+                settings = config.get('settings')
+                if isinstance(settings, str) and Path(settings).is_file():
+                    with open(settings, 'r') as sf:
```

Ruff (0.15.21)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)